pyaerocom/io/read_eea_aqerep.py

```diff
@@ -389,14 +426,13 @@ def get_station_coords(self, meta_key):
 
         Parameters
         ----------
-        meta_key : `str`
+        meta_key : str
             string with the internal station key
         """
-
         ret_data = {}
-        ret_data['latitude'] = self._metadata[meta_key][self.LATITUDENAME]
-        ret_data['longitude'] = self._metadata[meta_key][self.LONGITUDENAME]
-        ret_data['altitude'] = self._metadata[meta_key][self.ALTITUDENAME]
+        ret_data['latitude'] = float(self._metadata[meta_key][self.LATITUDENAME])
+        ret_data['longitude'] = float(self._metadata[meta_key][self.LONGITUDENAME])
+        ret_data['altitude'] = float(self._metadata[meta_key][self.ALTITUDENAME])
```

Comment:
np.float again

Reply:
float also works since it is not written into the numpy _data array at the moment, but is only in the metadata. but I can change if you want.